client/lib/js/react/stores/asset_store.js

```diff
@@ -81,6 +80,12 @@ class AssetStore {
         this.getInstance().transferAsset();
     }
 
+    onEscrowAsset({ idToTransfer, payloadToPost }) {
+        this.assetMeta.idToTransfer = idToTransfer;
+        this.assetMeta.payloadToPost = payloadToPost;
+        this.getInstance().escrowAsset();
```

Comment:
BTW, instead of using `assetMeta`, you can directly pass arguments to your source: `this.getInstance().escrowAsset(idToTransfer, payloadToPost)`.